### PR TITLE
Add project name and a link to project page in stage page.

### PR DESCRIPTION
**Pull request by euccas**
In Teletraan environment's stage page, add project name and a link to the project page.

- This change helps deployment operators to easily find project ownership information.
- Backend change: get project name from the query result of nimbus identifier.
- UI change: add a link next to the "Active Deployment" label.

teletraan_project_info_link

### CODE REVIEW

File: deploy-board/deploy_board/webapp/env_views.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 from django.contrib import messages
 from deploy_board.settings import IS_PINTEREST
 from deploy_board.settings import TELETRAAN_DISABLE_CREATE_ENV_PAGE, TELETRAAN_REDIRECT_CREATE_ENV_PAGE_URL,\
-    IS_DURING_CODE_FREEZE, TELETRAAN_CODE_FREEZE_URL, TELETRAAN_JIRA_SOURCE_URL, TELETRAAN_TRANSFER_OWNERSHIP_URL,TELETRAAN_RESOURCE_OWNERSHIP_WIKI_URL
+    IS_DURING_CODE_FREEZE, TELETRAAN_CODE_FREEZE_URL, TELETRAAN_JIRA_SOURCE_URL, TELETRAAN_TRANSFER_OWNERSHIP_URL, TELETRAAN_RESOURCE_OWNERSHIP_WIKI_URL,  TELETRAAN_PROJECT_URL_FORMAT
```
Comment (yongwen) [inline deploy-board/deploy_board/webapp/env_views.py]: looks like TELETRAAN_PROJECT_URL_FORMAT is not being used in this module

File: deploy-board/deploy_board/templates/environs/env_landing.html [hunk 2]
```diff
@@ -345,6 +345,12 @@ <h4 class="panel-title pull-left pointer-cursor">
                 {% endif %}
                 </a>
             </h4>
+            {% if project_info %}
+            <div class="btn-project-info">
+                <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+                <a href="{{ project_info.project_url }}" target="_blank">Project: {{ project_info.project_name }}</a>
+            </div>
+            {% endif %}
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/environs/env_landing.html]: can you display a warning something like "warning: no project ownership info, please contact slack #infra-console" in the case of no project. 
and group this block into IS_PINTEREST flag.

File: deploy-board/deploy_board/webapp/env_views.py (hunk 1 above)
Comment (euccas) [reply deploy-board/deploy_board/webapp/env_views.py]: right it's not used. i need delete it from this this module env_views.

File: deploy-board/deploy_board/templates/environs/env_landing.html (hunk 2 above)
Comment (euccas) [reply deploy-board/deploy_board/templates/environs/env_landing.html]: added the warning message when a project does not exist.

**Pull request merged**